Makefile

```diff
@@ -355,15 +362,19 @@ androidupdate: all
 	-rm $(INSTALL_DIR)/koreader/libs/libluajit.so
 
         # fresh APK assets
-	rm -rfv $(ANDROID_ASSETS)
-	mkdir -p $(ANDROID_ASSETS)
+	rm -rfv $(ANDROID_ASSETS) $(ANDROID_LIBS_ROOT)
+	mkdir -p $(ANDROID_ASSETS) $(ANDROID_LIBS_ABI)
 
 	# APK version
 	echo $(VERSION) > $(ANDROID_ASSETS)/version.txt
 
 	# shared libraries are stored as raw assets
 	cp -pR $(INSTALL_DIR)/koreader/libs $(ANDROID_LAUNCHER_DIR)/assets
 
+	# binaries are stored as shared libraries to prevent W^X exception on Android 10+
```

Comment:
W^X?

Reply:
https://en.m.wikipedia.org/wiki/W%5EX

Reply:
Maybe add a link to documentation?: https://developer.android.com/about/versions/10/behavior-changes-10#execute-permission

Reply:
Yes please! :-)